src/components/Chat/RemoteVideoContainer.js

```diff
@@ -34,11 +33,12 @@ class RemoteVideoContainer extends React.Component {
   }
 
   addPeerVideos() {
+    let riffIds = [...this.props.chat.webRtcRiffIds].sort();
     let peerLength = this.props.peers.length;
     logger.debug("rendering", peerLength, "peers....", this.props.peers);
     logger.debug("names:", this.props.chat.webRtcPeerDisplayNames);
     logger.debug("riff ids:", this.props.chat.webRtcRiffIds);
-    return this.props.peers.map(this.peerVideo(peerLength));
+    return this.props.peers.map(this.peerVideo(peerLength,riffIds));
```

Comment:
style: comma separated lists should have a space after the comma
```suggestion
    return this.props.peers.map(this.peerVideo(peerLength, riffIds));
```